### PR TITLE
fix: Updated react-popper version

### PR DESCRIPTION
**Pull request by greg-a-smith**
Updating react-popper to 0.8.3.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,7 +107,8 @@
       "cat coverage/*/lcov.info | ./node_modules/codecov.io/bin/codecov.io.js",
     "prebuild": "rimraf es lib dist",
     "build": "cross-env NODE_ENV=production run-p build:** && run-p css:**",
-    "build-dev": "cross-env NODE_ENV=development run-p build:** && run-p css:**",
```
Comment (martijnrusschen) [inline package.json]: This change seems not intentional

Comment (greg-a-smith) [reply package.json]: Agreed -- not sure where that came from.

Comment (greg-a-smith) [reply package.json]: Okay, what's going on here?  I went back to explicitly revert this and it showed it as reverted, but after committing and pushing it, it keeps changing it to this format.

Comment (martijnrusschen) [reply package.json]: Looks like both are the same now. I'm fine with this.

**Pull request merged**